tests/meta.yaml

```diff
@@ -0,0 +1,161 @@
+# yamllint disable rule:line-length
+domains:
+  - name: domain.com
+    type: sssd
+    hosts:
+      - name: client
+        external_hostname: client
+        # Default to get flavour with 2GB RAM that is required for mssql-server
```

Comment:
"flavor" is more common...? :)

Reply:
AmE against BrE :) Red Hat goes with AmE, so my behaviour is not of a accepted flavour :) Actually, this meta.yaml file will be present in IDMCI, so I'll remove it from this PR